network/peer/test_peer.go

```diff
@@ -117,6 +118,7 @@ func StartTestPeer(
 			MaxClockDifference:   time.Minute,
 			ResourceTracker:      resourceTracker,
 			IPSigner:             NewIPSigner(signerIP, tls),
+			UptimeCalculator:     uptime.NoOpCalculator,
```

Comment:
Super small nit, but can we put the `UptimeCalculator` between `ResourceTracker` and `IPSigner`? That's how it's defined in the `Config`, so I think it's easier to make sure we're populating the expected fields if we keep them in the same order.